modules/installation-disk-partitioning-upi-templates.adoc

```diff
@@ -35,6 +35,51 @@ Because `/var` must be in place before a fresh installation of {op-system-first}
 If you follow the steps to create a separate `/var` partition in this procedure, it is not necessary to create the Kubernetes manifest and Ignition config files again as described later in this section.
 ====
 
+.Prerequisites
+
+* If using a ContainerRuntimeConfig, and the container storage is on the root partition, ensure that this root partition includes the `pquota` mount option.
```

Comment:
@EricPonvelle hi, considering this section is for `Creating a separate /var partition `, so I think we don't need to mention `the container storage is on the root partition `, wdyt?

And also according to https://access.redhat.com/solutions/5639371, `If the container storage is on the root partition make sure that the root partition is mounted with the pquota option by having rootflags=pquota in the grub command line as per the RHEL solution article`

The following examples are for the case when `If the container storage is on a separate partition `